prob/torus_cbc/problem.c

```diff
@@ -471,7 +471,8 @@ void init_prob() {
 
       q = P[i][j][k][RHO] / rhomax - 0.2;
 
-      P[i][j][k][B3] = (q > 0.) ? q : 0.0;
+      // P[i][j][k][B3] = (q > 0.) ? q : 0.0;
+      P[i][j][k][B3] = (q > 0.) ? 1 : 0.0;
```

Comment:
Hard to complain about this (except maybe that one could add a comment on the commented-out line like `// oddly, this leads to instability`?) Up to you, feel free to merge now if you want

Reply:
I'm just gonna merge it in. YOLO.